packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -132,7 +132,7 @@ function hookFunctionsPush(): IWorkflowExecuteHooks {
 				if (this.sessionId === undefined) {
 					return;
 				}
-				LoggerProxy.verbose(`Executing hook nodeExecuteBefore for hookFunctionsPush on node ${nodeName}`, {executionId: this.executionId});
+				Logger.verbose(`Executing hook on node "${nodeName}" (nodeExecuteBefore, hookFunctionsPush)`, {executionId: this.executionId});
```

Comment:
why not include workflowId in logs in this file too? as well as sessionId? would make debugging that much easier.